### ISSUE TITLE
Change default color scheme for pre tags

### ISSUE THREAD

**Issue by apohllo** [apache/airflow#32695]
### Description

The default color scheme for pre elements:
![image](https://github.com/apache/airflow/assets/40543/65c71b31-4275-4540-9416-91d3b6fa721f)
is not very convenient. Grey letters on grey background are hard to read. 
Since these elements are styled as fixed-font, I don't think it is necessary to provide these additional styling features.
I think keeping the color black, and the background white would provide much better UX.

### Use case/motivation

This is just my first impressiont regarding AirFlow. I am looking into the code of example tasks and I am looking into the log of these tasks, trying to find a particular piece of information. The first impression is that it is very hard for my eyes to read the default color scheme.

I can apply a custom CSS to change that color scheme, but I think there are many other people that have the same problem.

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by pankajkoti** [comment]
cc: @pierrejeambrun @bbovenzi ☝🏽

**Comment by bbovenzi** [comment]
Makes sense to me! That is the legacy logs view. How do the `pre` elements look to you in the grid view logs?

**Comment by bbovenzi** [comment]
Pre tags in Airflow 3 are more readable

<img width="843" alt="Image" src="https://github.com/user-attachments/assets/8299c170-7ec3-49c1-ad7e-117e8916c409" />

<img width="925" alt="Image" src="https://github.com/user-attachments/assets/ded385fd-a6fb-494e-acb7-01d2802ce938" />
